community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -181,7 +181,7 @@ def create_instances_request(nodes: List[str], placement_group: Optional[str], e
         ),
     )
 
-    if placement_group:
+    if placement_group and excl_job_id is not None:
```

Comment:
Curious what the motivation is for this, we don't want to enforce "all or nothing" provisioning for job associated requests?

Reply:
one needs to check changes before committing , thank you for spotting.